cdap-client/src/main/java/co/cask/cdap/client/ArtifactClient.java

```diff
@@ -564,6 +565,28 @@ public void delete(Id.Artifact artifactId) throws IOException, UnauthenticatedEx
   }
 
   /**
+   * Delete all artifacts in the specified namespace.
+   *
+   * @param namespaceId the namespace to delete all artifacts from
+   *
+   * @throws BadRequestException if the request is invalid. For example, if the artifact name or version is invalid
+   * @throws IOException if a network error occurred
+   * @throws UnauthenticatedException if the request is not authorized successfully in the gateway server
+   */
+  public void delete(NamespaceId namespaceId) throws IOException, UnauthenticatedException, BadRequestException {
```

Comment:
Good catch! I don't think we need this right now. I may have missed cleaning this up. Will remove.